Blish HUD/GameServices/Gw2MumbleService.cs

```diff
@@ -50,6 +50,21 @@ public class Gw2MumbleService : GameService {
 
         #endregion
         
+        #region Events
+        /// <summary>
+        /// Fires when the mumble availability changes.
+        /// </summary>
+        public event EventHandler<ValueEventArgs<bool>> IsAvailableChanged;
+
+        private void OnIsAvailableChanged(ValueEventArgs<bool> e) => IsAvailableChanged?.Invoke(this, e);
+
+        private bool   _prevIsAvailable               = false;
```

Comment:
Get rid of the extra spacing here and put a newline before the `#endregion` and after the `#region Events` portions and we should be good.  👍